src/Styling/Home.css

```diff
@@ -1,7 +1,75 @@
+/* font styles */
+.style_1 {
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 6.25rem;
+  color: #DE5B31;
+}
+#style_1 {
+  color: transparent;
+  -webkit-text-stroke: 1px #fff;
+}
+.style_2 {
+  font-family: 'Roboto Mono';
+  font-weight: 400;
+  font-size: 17px;
+  color: #fff;
+}
 /* Hero section */
+.Hero_section {
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  position: relative;
+  grid-template-columns: 10% 1fr 2fr 10%;
+  grid-template-rows: 10% 1fr 1fr 10%;
+}
 .Hero_image {
-    width: 100vw;
-    height: 100vh;
-    background-repeat: no-repeat;
-    background-size: cover;
+  width: 100vw;
+  height: 100vh;
+  position: absolute;
+  z-index: -1;
+}
+.Hero_overlay {
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 150px;
+  grid-row: 2 / span 2;
+  grid-column: 3;
+}
+.Hero_overlay :nth-child(2) {
+  text-align: center;
+  margin-top: 20px;
+  line-height: 1;
+}
+.Hero_overlay :nth-child(3) {
+  text-align: right;
+  margin-top: 20px;
+}
+
```

Comment:
Consistency in spacing is just as important as following a consistent naming convention. Either add an empty line between each style class/id like you did here, or have no empty lines in between each class/id